apis/falcon/v1alpha1/falconconfig_types.go

```diff
@@ -17,13 +17,28 @@ type FalconAPI struct {
 	ClientSecret string `json:"client_secret"`
 }
 
+// LinuxContainerSpec configures Falcon Container Sensor product installation on your cluster
+type LinuxContainerSpec struct {
+	Enabled bool `json:"enabled"`
+	// Container Registry to which falcon-operator will push falcon-container sensor and from which falcon-container sensor will be consumed by pods
+	Registry string `json:"registry"`
+}
+
+// WorkloadProtectionSpec configures workload protection on the cluster
+type WorkloadProtectionSpec struct {
```

Comment:
@isimluk can this be renamed to `ContainerProtectionSpec` or if the Falcon kernel sensor is going under this as well to be `FalconProtectionSpec`?

Reply:
my thinking was Workload protection vs static analysis

Anyway, I think this is by no means final design, it is just the first data model, that will have to be redesigned couple times as we go and learn.

Reply:
It may happen that we will have to extract each piece to separate CRD. I don't know that yet for sure.

Reply:
I mean this is an initial start-ing point to get the ball rolling...

Reply:
I guess you are right... :D